credhub-format.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -eo pipefail
+
+# yq operates on yaml documents using a jq-like syntax.
+# For jq program syntax, see: https://stedolan.github.io/jq/manual/
+
+if [ -z $2 ]; then
```

Comment:
Since the args are positional, if you pass only zero or one args, this will evaluate to false. And if you pass two, you're golden. That said, checking the count is probably more clear and conventional. I'll switch to that.